src/components/EmojiReactions/EmojiReactions.tsx

```diff
@@ -191,86 +212,100 @@ const EmojiReactions = () => {
 
   return (
     <div className="p-6">
-      <h1>Emoji Reactions</h1>
-      <p>
-        Hello to you and welcome to Ably. You can react to any of these
-        messages.
-      </p>
-      <p>Go ahead, give it a try! You can always checkout the source code </p>
+      <div>
+        <h1 className="text-xl mb-5">Emoji Reactions</h1>
+        <p>
+          Hello to you and welcome to Ably. You can react to any of these
+          messages.
+        </p>
+        <p>Go ahead, give it a try! You can always checkout the source code </p>
+      </div>
       {/* Display default chat message */}
```

Comment:
Comments don't need the `{ }` braces, could we remove them so it's consistent across the repo? :)
Also if we add a line break above the comments would make it easier to read.